src/world.rs

```diff
@@ -429,31 +447,103 @@ impl LazyInsertions {
     /// struct InsertPos;
     ///
     /// impl<'a> System<'a> for InsertPos {
-    ///     type SystemData = (Entities<'a>, Fetch<'a, LazyInsertions>);
+    ///     type SystemData = (Entities<'a>, Fetch<'a, LazyUpdate>);
     ///
     ///     fn run(&mut self, (ent, lazy): Self::SystemData) {
     ///         let a = ent.create();
     ///         let b = ent.create();
     ///
-    ///         lazy.add(vec![(a, Pos(3.0, 1.0)), (b, Pos(0.0, 4.0))]);
+    ///         lazy.insert(vec![(a, Pos(3.0, 1.0)), (b, Pos(0.0, 4.0))]);
     ///     }
     /// }
     /// ```
-    pub fn add<L>(&self, l: L)
+    pub fn insert<L>(&self, l: L)
```

Comment:
Yes. I'm not sure, however, if executing `call_once` on `Box<FnOnce` is permitted / if there's a workaround.